src/request/import/ImportWords.js

```diff
@@ -31,8 +31,8 @@ class ImportWords {
         /** @type {{entropy: Nimiq.Entropy?, privateKey: Nimiq.PrivateKey?}} */
         this._secrets = { entropy: null, privateKey: null };
         // eslint-disable-next-line max-len
-        /** @type {{entropy: Partial<KeyguardRequest.SingleKeyResult>, privateKey: Partial<KeyguardRequest.SingleKeyResult>}} */
-        this._keyResults = { entropy: {}, privateKey: {} };
+        /** @type {KeyguardRequest.KeyResult} */
+        this._keyResults = [];
```

Comment:
should be renamed to singular form then probably

Reply:
or maybe not... I don't know.

Reply:
It is 1 or 2, so i think results is fine.